javascript-modal/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Javascript Modal</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <button type="button" class="openButton">Open Modal</button>
+    </div>
+
+    <div class="container">
+      <div class="surveryQuestionPopupWindow overlay hidden">
+        <div class="popup">
+          <h1 class="surveyQuestion">Would you like to take a survey?</h1>
```

Comment:
Class names should be using `kebab-cased`, not `camelCased`

Reply:
Updated